Makefile

```diff
@@ -5,13 +5,15 @@ SHELL=/bin/bash -o pipefail
 build:
 	@docker pull heroku/heroku:18-build
 	@docker build -f Dockerfile.build -t heroku/pack:18-build .
+	@docker build -f Dockerfile.riff.build -t heroku/pack:18-riff-build .
 	@docker build -f Dockerfile.run -t heroku/pack:18 .
 	@pack create-builder heroku/buildpacks:18 --builder-config builder.toml --no-pull
 	@pack create-builder heroku/functions-buildpacks:18 --builder-config functions-builder.toml --no-pull
 	@docker tag heroku/buildpacks:18 heroku/buildpacks:latest
 	@docker tag heroku/functions-buildpacks:18 heroku/functions-buildpacks:latest
 
 publish: build
+	@docker push heroku/pack:18-riff-build
```

Comment:
What's the reasoning for adding another image? This seems like a lightweight change that would make sense in `pack:18-build` and probably be fine for `pack:18` for that matter.

Reply:
@joshwlewis I'm being paranoid, maybe too much.  I could be swayed to add this to the exiting `pack:18-build` we if don't think it would cause any issues having `RIFF=1` in all of the images.  It should only be for a short amount of time.

Reply:
Why only a short amount of time? Are there upstream changes coming?

Reply:
I'd rather not have it in the `heroku/pack:18-build` image. There could be unforeseen consequences to people using `heroku/buildpacks:18`. Image layers are cheap (especially this one) so I don't mind the extra tag if need be. I haven't thought it out well though.

Reply:
@joshwlewis yeah, Ben Hale is planning something that should allow this to go away.  Once that mechanism is in place we should be able to remove this.

Reply:
This just sets `RIFF=1` for every image. I don't think this would cause issues for most people, but probably better to be safe atm than sorry.

Reply:
Yeah, I think I lean toward the extra image for safety + it will ensure we keep poking the riff team to get it moved to `requires` for consumers like us. 

Just thinking out loud...root buildpacks wouldn't allow us to fix this either right? Since the env var needs to be done prior to `detect`.

Reply:
Yeah, this is required prior to `detect`.